package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "platform-custom-scalars",
+  "version": "1.0.0",
+  "private": false,
+  "description": "A library of customized GraphQL scalars for Ontotext Platform",
+  "homepage": "https://github.com/Ontotext-AD/platform-custom-scalars#readme",
```

Comment:
Also add a license like:
```
"license": "Apache-2.0",
```
and the actual license like this I guess
https://github.com/Ontotext-AD/graphdb.js/blob/master/LICENSE

Reply:
Yes, I have TODO about that. I'll include that information.

Reply:
Done.